logging/src/main/java/org/apache/rocketmq/logging/dynamic/AbstractProcessUnitImpl.java

```diff
@@ -0,0 +1,116 @@
+package org.apache.rocketmq.logging.dynamic;
+
+import com.alibaba.fastjson.JSONArray;
+import com.alibaba.fastjson.JSONObject;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.impl.StaticLoggerBinder;
+
+import java.util.Enumeration;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import static org.apache.rocketmq.logging.dynamic.LogConstant.LOG4J2_LOGGER_FACTORY;
+import static org.apache.rocketmq.logging.dynamic.LogConstant.LOG4J_LOGGER_FACTORY;
+import static org.apache.rocketmq.logging.dynamic.LogConstant.LOGBACK_LOGGER_FACTORY;
+import static org.apache.rocketmq.logging.dynamic.LogConstant.LOGGER_LEVEL;
+import static org.apache.rocketmq.logging.dynamic.LogConstant.LOGGER_LIST;
+import static org.apache.rocketmq.logging.dynamic.LogConstant.LOGGER_NAME;
+import static org.apache.rocketmq.logging.dynamic.LogConstant.LOGGER_TYPE_UNKNOWN;
+import static org.apache.rocketmq.logging.dynamic.LogConstant.LOG_FRAMEWORK;
+import static org.apache.rocketmq.logging.dynamic.LogConstant.ROOT_KEY;
+
+/**
+ * 日志调整抽象类
+ * 支持log4j、log4j2、logback
+ */
```

Comment:
remove chinese notes and hints,Other classes also have Chinese annotations. don't have a valid license header.
Code guidelines: https://rocketmq.apache.org/docs/%E8%B4%A1%E7%8C%AE%E6%8C%87%E5%8D%97/30code-guidelines